mm2src/mm2_bitcoin/chain/src/block_header.rs

```diff
@@ -234,7 +234,8 @@ impl Deserializable for BlockHeader {
             None
         };
 
-        let is_zcash = version == 4 && !reader.coin_variant().is_btc() && !reader.coin_variant().is_ppc();
+        let is_zcash =
+            version == 4 && !reader.coin_variant().is_btc_or_ppc() || reader.coin_variant().is_morty_or_rick();
```

Comment:
I think better to add `is_kmd_assetchain` instead of `is_morty_or_rick`. Even if we decide not to check the coins config, it could be easier to hardcode some asset chains.

Reply:
done